### PR TITLE
SITES-681 Image asset listing and liquid image tag

### PR DESCRIPTION
**Pull request by maxious**

### CODE REVIEW

File: config/initializers/kaminari_config.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Kaminari.configure do |config|
+  # config.default_per_page = 25
```
Comment (ltankey) [inline config/initializers/kaminari_config.rb]: Do we need any of these default configuration items?

File: app/controllers/editorial/assets_controller.rb [hunk 2]
```diff
@@ -18,8 +18,8 @@ def create
         asset = @form.model
         asset.uploader = current_user
         if asset.save
-          flash[:notice] = "Asset created, available at https:#{asset.asset_file.url}"
-          redirect_to editorial_root_path and return
+          flash[:notice] = "Asset successfully created"
+          redirect_to editorial_assets_path and return
```
Comment (ltankey) [inline app/controllers/editorial/assets_controller.rb]: The tests probably need to be updated to reflect this.

File: config/initializers/kaminari_config.rb (hunk 1 above)
Comment (maxious) [inline config/initializers/kaminari_config.rb]: Nope. You can generate the config file later but I thought better to have something you can grep for.

File: app/controllers/editorial/assets_controller.rb (hunk 2 above)
Comment (maxious) [inline app/controllers/editorial/assets_controller.rb]: Done.

**Pull request merged**